Add handling of missing devices to HomematicIP Cloud (#11725)
home-assistant/home-assistant.github.io: source/_integrations/homematicip_cloud.markdown

  * Weather Sensor – plus (*HmIP-SWO-PL*)
  * Weather Sensor – pro (*HmIP-SWO-PR*)
  

> ## What to do, if a device is missing in Home-Assistant

Reviewer: ```suggestion
## What to do, if a device is missing in Home Assistant
```

> In order for a device to be integrated in the Home-Assistant, this must first be implemented in the upstream library. A dump of your configuration is required for this, which is then attached to a new issue in the [upstream lib's](https://github.com/coreGreenberet/homematicip-rest-api) github repository.

Reviewer: ```suggestion
In order for a device to be integrated into Home Assistant, it must first be implemented in the upstream library. A dump of your configuration is required for this, which is then attached to a new issue in the [upstream lib's](https://github.com/coreGreenberet/homematicip-rest-api) GitHub repository.
```

> 1.	Create a dump of your access point configuration in Home-Assistant: 

Reviewer: ```suggestion
1. Create a dump of your access point configuration in Home Assistant: 
```

> Developer Tools -> Services -> Select `homematicip_cloud.dump_hap_config` -> Execute. 

Reviewer: ```suggestion
  Developer Tools -> Services -> Select `homematicip_cloud.dump_hap_config` -> Execute. 
```

> The default dump is anonymized and is written to your config directory (`hmip_config_XXXX.json`)

Reviewer: ```suggestion
  The default dump is anonymized and is written to your configuration directory (`hmip_config_XXXX.json`).
```

> 2.  Create a [new issue](https://github.com/coreGreenberet/homematicip-rest-api/issues/new) in this github repository and attach the dump file.

Reviewer: ```suggestion
2. Create a [new issue](https://github.com/coreGreenberet/homematicip-rest-api/issues/new) at this GitHub repository and attach the created dump file.
```

> Please be patient and wait for the implementation and a new release of the upstream library.

Reviewer: ```suggestion
Please be patient, wait for the implementation and a new release of the upstream library.
```

> Afterwards this device can be implemented in Home-Assistant.

Reviewer: ```suggestion
Afterward, this device can be implemented into Home Assistant.
```
